### PR TITLE
Big Paperwork: Adds the Paperwork loan offer, paperwork fraud, and ancient paperwork!

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

Adds paperwork -- an indecipherable mess of papers that only a seasoned department head could hope to decipher. If processed and sent to Central Command via the cargo shuttle, the budget gets a bonus.

Paperwork is distributed through a new Shuttle Loan offer. A nearby station needs some paperwork reviewed. Do you have what it takes to handle BIG PAPERWORK?

Processing paperwork is relatively simple. You can inspect it to see which head of staff it "belongs" to, and you can use their stamp to complete it. You get a lot of these, so the bonuses can add up if you get them all in. Just be careful not to return any unprocessed paperwork.

With big paperwork comes big paperwork fraud, and big fines.

image

You can photocopy your completed paperwork to double-dip with Central Command and squeeze out a little bit more money. The more paperwork fraud you commit, the higher the chance you'll be caught and fined.

On top of that:

tboi unlock

Some say that in the long forgotten halls of maintenance, there lies _ancient paperwork_. Documents whose importance has long since passed, but are still important for bookkeeping at Central Command. With a keen eye, you might spot one of these while scouring through maintenance. Make sure it gets forwarded to the right person, cargo will thank you.

## Why It's Good For The Game

More shuttle loans offers means more event variation.

Provides convenient access to the bottom layer of hell (Fraud) for cargo staff. 

Adds some rare maintenance loot that, with a tiny bit of effort, benefits the crew. It's like a side quest for assistants.

## Changelog

:cl:
add: Paperwork! Huge paperwork. Gigantic. You can stamp it and send it to Centcom via cargo for BIG MONEY.
add: New shuttle loan event variant -- You get a bunch of paperwork!
add: Ancient paperwork has appeared in the maintenance tunnels!
/:cl:

### CODE REVIEW

**Comment by private-tristan** [comment]
The Binding of Spessmen

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by GuiltyNeko** [comment]
I love this it's absolutely fantastic, please merg.
Also I noticed that these all have the same first detailed description line, is it possible to do code magic to just write that once and have it apply to all of them to reduce the line count? Or would that be too "point A and point B too far separated from each other and now confusing"?

**Comment by Jackal-boop** [comment]
Holy shit rhials

**Comment by bruhman8ghughug** [comment]
i love when stuff in the lore (tax fraud) gets reflected on gameplay

**Review by Twaticus — CHANGES_REQUESTED** [review]
chrome_Yr860DJIig

**Review by Twaticus — APPROVED** [review]

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Very small things but looks good to merge after

File: code/modules/events/shuttle_loan.dm [hunk 1]
```diff
@@ -277,6 +287,11 @@
 				else
 					shuttle_spawns.Add(/obj/item/paper/fluff/cargo/bomb/allyourbase)
 
+			if(PAPERS_PLEASE)
+				for(var/new_paperwork in subtypesof(/obj/item/paperwork))
+					if(!ispath(new_paperwork, /obj/item/paperwork/photocopy) && !ispath(new_paperwork, /obj/item/paperwork/ancient))
+						shuttle_spawns.Add(new_paperwork)
```
Comment (Mothblocks) [inline code/modules/events/shuttle_loan.dm:293]:
```suggestion
				shuttle_spawns += subtypesof(/obj/item/paperwork) - typesof(/obj/item/paperwork/photocopy) - typesof(/obj/item/paperwork/ancient)
```

File: code/modules/paperwork/paperwork.dm [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+/obj/item/paperwork/service
+	stamp_requested = /obj/item/stamp/hop
+	stamp_job = /datum/job/head_of_personnel
+	stamp_icon = "paper_stamp-hop"
+
+/obj/item/paperwork/service/Initialize()
+	. = ..()
+
+	detailed_desc += span_info(" Your begin scanning over the document. This is a standard Nanotrasen NT-435Z3 form used for requests to Central Command.")
```
Comment (GuiltyNeko) [inline code/modules/paperwork/paperwork.dm:140]: Your Begin

**Comment by DAKKA-WAAAGH** [comment]
YEAAAAHHH!!

**Pull request merged**